app/start_handlers.py

```diff
@@ -160,7 +161,7 @@ async def post(self, request):
                 raise ex
 
         if uac_json['caseId'] is None:
-            logger.info('unlinked case', client_ip=request['client_ip'])
+            logger.info('unlinked case', client_ip=request['client_ip'], case_details=uac_json, region_of_site=display_region)
```

Comment:
Is it safe to log the whole case return?

Reply:
Yes, I think you're right, I wasn't sure when I added it. I think I should remove it so I'll do that.